snorkel/learning/pytorch/logistic_regression.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from builtins import *
+
+import torch.nn as nn
+from scipy.sparse import issparse
+
+from snorkel.learning.pytorch import TorchNoiseAwareModel
+
+
+class LogisticRegression(TorchNoiseAwareModel):
+    representation = False
+    
+    def __init__(self, input_size, num_classes):
+        super(LogisticRegression, self).__init__()
+        self.linear = nn.Linear(input_size, num_classes)
+        self.sigmoid = nn.Sigmoid()
+
+    def forward(self, X):
+        out = self.linear(X)
+        return self.sigmoid(out)
+        
+    def _check_input(self, X):
+        if issparse(X):
+            raise Exception("Sparse input matrix. Use SparseLogisticRegression")
```

Comment:
Since there's no PyTorch version of `SparseLogisticRegression` implemented here yet, just cast to dense and print a warning about this?

Reply:
Done 75dc24a